sources/packages/web/src/services/UserService.ts

```diff
@@ -23,13 +27,15 @@ export class UserService {
     }
   }
 
-  async getBCeIDAccounts(
-    authHeader?: any,
-  ): Promise<BCeIDAccountsDto | null> {
+  async getBCeIDAccounts(authHeader?: any): Promise<BCeIDAccountsDto | null> {
     try {
       return await ApiClient.User.bceidAccounts(authHeader);
     } catch (excp) {
       return null;
     }
   }
+
+  async getAllUserLocations(authHeader?: any): Promise<UserLocationDto[]> {
```

Comment:
This method does not need to support the custom header parameter.
The methods that have that are methods that need to be called too early in the process when the authentication token is not even in place and accessible globally.

Reply:
Done